### PR TITLE
CIRCLE-6587 - Nomad operation guide for enterprise

### PR DESCRIPTION
**Pull request by kimh**

### CODE REVIEW

**Comment by ndintenfass** [comment]
We should probably add some information about how to get the nomad CLI and how to configure it to talk to CircleCI. Or is the assumption that people are only doing this by SSHing to the services box and running from there? Is that the _only_ way to do it, or can people use the nomad CLI externally and point at CircleCI?

**Comment by kimh** [comment]
@ndintenfass I added a line to say where to run these commands

File: jekyll/_ccie/nomad.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+Nomad Job is a specification provided by users that declares a workload for Nomad. In CircleCI 2.0, a Nomad job corresponds to an execution of CircleCI job/build. If the job/build uses parallelism, say 10 parallelism, then Nomad will run 10 jobs.
+
+### Build Agent
+Build Agent is a Go program written by CircleCI that executes steps in a job and reports the results. Build Agent is executed as the main process inside a Nomad Job.
+
+# Basic Operations
+
+This section will give you the basic guide of operating a Nomad cluster in your installation.
+
+You can use Nomad CLI in both Nomad servers and clients but we recommend running the following commands in this section from one of your Nomad clients.
```
Comment (ndintenfass) [inline jekyll/_ccie/nomad.md]: Still fear the uninitiated won't know how to configure their cli to talk to the right thing. They point at the ip of the services box? How is authentication done?

**Comment by kimh** [comment]
@ndintenfass I'm merging this now but your feedbacks/wordsmithing is very welcome.

**Comment by kimh** [comment]
I'll close this as https://github.com/circleci/circleci-docs/pull/1511 is created under the new layout

**Pull request closed without merging**